docs/JestObjectAPI.md

```diff
@@ -715,6 +754,56 @@ Returns the `jest` object for chaining.
 
 Restores all mocks back to their original value. Equivalent to calling [`.mockRestore()`](MockFunctionAPI.md#mockfnmockrestore) on every mocked function. Beware that `jest.restoreAllMocks()` only works when the mock was created with `jest.spyOn`; other mocks will require you to manually restore them.
 
+
+### `jest.mocked<T>(item: T, deep = false)`
```

Comment:
`jest.mocked` is already documented. Why is this needed?

Reply:
This is a variation on jest.mocked and was preexisting from the Docs I forked.

Reply:
Since Jest 29 this method has [different API](https://jestjs.io/docs/mock-function-api#jestmockedsource-options). Documentation got reworked accordingly. I don’t see why outdated documentation should be added, if the goal of your PR is to add missing admonitions. That it definitely out of scope. If there are any issues with current documentation of `jest.mocked`, please open a separate PR.